uhabits-android/src/main/java/org/isoron/uhabits/activities/common/views/FrequencyChart.kt

```diff
@@ -222,15 +224,16 @@ class FrequencyChart : ScrollableChart {
         canvas.drawLine(rGrid.left, rGrid.top, rGrid.right, rGrid.top, pGrid!!)
     }
 
-    private fun drawMarker(canvas: Canvas, rect: RectF?, value: Int?) {
+    private fun drawMarker(canvas: Canvas, rect: RectF?, value: Int?, frequency: Int) {
         // value can be negative when the entry is skipped
         val valueCopy = value?.let { max(0, it) }
 
         val padding = rect!!.height() * 0.2f
         // maximal allowed mark radius
         val maxRadius = (rect.height() - 2 * padding) / 2.0f
         // the real mark radius is scaled down by a factor depending on the maximal frequency
-        val scale = 1.0f / maxFreq * valueCopy!!
```

Comment:
Do we still need `maxFreq`?

Reply:
I don't think so, it was only used in the previous drawMarker method and on an unused method populateWithRandomData. We could also get rid of the getMaxFrequency method.

I just wanted to make sure that you prefer this new approach of showing the percentage rather than the frequency, because this may only be useful for everyday habits were the actual day of the week is meaningful.